frontend/lab/samples/loops/gradient-show.htmlf

```diff
@@ -0,0 +1,49 @@
+<h1>🌈📺 Gradient</h1>
+
+<p>Run the program to draw an animated colour gradient.</p>
```

Comment:
You have used the British spelling of colour here, where you always use the American spelling elsewhere.